### ISSUE TITLE
 怎样设置测试集数据文件夹？ How Do I set up the test set data folder?

### ISSUE THREAD

**Issue by JoshuaYang0926** [jeya-maria-jose/UNeXt-pytorch#18]
I ran into some problems running the test val. PY file! How Do I set up my training data?
